### DISCUSSION TITLE
Hány vevőnél lehet ugyanaz az eNyugta?

### DISCUSSION THREAD

**Discussion by Worksdot** [nav-gov-hu/eRECEIPT#25] [Kérdések és válaszok]
Engedélyezett működés-e, hogy egy eladó által prezentált QR kódot több vevő is beolvas a vásárlói alkalmazásába? Vagyis lehet-e, hogy egy eNyugta több vásárlónál is letöltésre kerül?

**Comment by renced42** [comment]
Kedves @Worksdot 
Jelenleg nem tiltott az elképzelések alapján, de ez még mozoghat. Elég kevés az eset, amikor erre sor kell, hogy kerüljön, de lehetséges ilyen. Inkább azt mondanám, hogy a enyugta megosztása jöhet szóba.
